test/verify/test_roialign.cpp

```diff
@@ -44,20 +77,23 @@ struct test_roialign : verify_program<test_roialign<DType>>
         auto x   = mm->add_parameter("x", x_s);
         auto roi = mm->add_parameter("roi", roi_s);
         auto ind = mm->add_literal(migraphx::literal(ind_s, ind_vec));
-        auto r   = mm->add_instruction(migraphx::make_op("roialign",
-                                                         {{"spatial_scale", 1.0},
-                                                          {"output_height", 5},
-                                                          {"output_width", 5},
-                                                          {"sampling_ratio", 2}}),
-                                     x,
-                                     roi,
-                                     ind);
+        auto r   = mm->add_instruction(
+            migraphx::make_op("roialign",
+                                {{"spatial_scale", 1.1},
+                                 {"output_height", 5},
+                                 {"output_width", 2},
+                                 {"sampling_ratio", 2},
+                                 {"coordinate_transformation_mode", "output_half_pixel"}}),
+            x,
+            roi,
+            ind);
         mm->add_return({r});
 
         return p;
     }
 };
 
+template struct test_roialign_half_pixel<migraphx::shape::float_type>;
```

Comment:
Is there any point in adding other data types for the half_pixel (i.e. the "real" ROIAlign) test?